packages/components/skeleton/src/skeleton.stories.ts

```diff
@@ -80,115 +78,74 @@ export const API: Story = {
 
         .container-images sl-skeleton {
           block-size: 8rem;
-          border-radius: 0.5rem;
         }
       </style>
-      <section>
+      <section aria-label="Loading">
         <div class="container">
           <div class="title">
-            <sl-skeleton .effect=${effect} aria-label="loading avatar"></sl-skeleton>
-            <sl-skeleton .effect=${effect} aria-label="loading header"></sl-skeleton>
+            <sl-skeleton variant="circle" .effect=${effect}></sl-skeleton>
+            <sl-skeleton .effect=${effect}></sl-skeleton>
           </div>
           <div class="body">
-            <sl-skeleton .effect=${effect} aria-label="loading paragraph"></sl-skeleton>
-            <sl-skeleton .effect=${effect} aria-label="loading paragraph"></sl-skeleton>
-            <sl-skeleton .effect=${effect} aria-label="loading paragraph"></sl-skeleton>
+            <sl-skeleton .effect=${effect}></sl-skeleton>
+            <sl-skeleton .effect=${effect}></sl-skeleton>
+            <sl-skeleton .effect=${effect}></sl-skeleton>
           </div>
         </div>
       </section>
-      <section>
+      <section aria-label="Loading pictures">
         <div class="container-images">
-          <sl-skeleton .effect=${effect} aria-label="loading picture"></sl-skeleton>
-          <sl-skeleton .effect=${effect} aria-label="loading picture"></sl-skeleton>
-          <sl-skeleton .effect=${effect} aria-label="loading picture"></sl-skeleton>
-          <sl-skeleton .effect=${effect} aria-label="loading picture"></sl-skeleton>
-          <sl-skeleton .effect=${effect} aria-label="loading picture"></sl-skeleton>
+          <sl-skeleton .effect=${effect}></sl-skeleton>
+          <sl-skeleton .effect=${effect}></sl-skeleton>
+          <sl-skeleton .effect=${effect}></sl-skeleton>
+          <sl-skeleton .effect=${effect}></sl-skeleton>
+          <sl-skeleton .effect=${effect}></sl-skeleton>
         </div>
       </section>
     `
 };
 
 export const All: Story = {
   args: {
-    width: '80%',
-    height: '0.75rem',
-    radius: '0.5rem',
-    background: '#d5d5d5',
-    shineColor: '#bcbcbc'
-  },
-  argTypes: {
-    background: {
-      control: {
-        type: 'color'
-      }
-    },
-    shineColor: {
-      control: {
-        type: 'color'
-      }
-    }
+    width: '80%'
   },
-  render: ({ width, height, radius, background, shineColor }) =>
+  render: ({ width }) =>
     html`
       <style>
         section {
           display: flex;
           flex-direction: column;
           gap: 0.5rem;
         }
+
+        sl-skeleton {
+          height: 1rem;
+        }
       </style>
-      <section>
+      <section aria-label="Loading">
```

Comment:
I don't think this is really necessary. :D Since there is already some text in this container, technically, it isn't loading :D 
I think the aria-label that has a value with "loading" in it should not have any text at all in it. If you have a mix of loaded and still loading elements you should make a deeper wrapper so that it contains only "loading" elements.

(but of course this is a weird example that no CFA will ever use in this way.)

Reply:
so I'll just remove `aria-label` from this story :)